norse/task/correlation_experiment.py

```diff
@@ -91,9 +91,9 @@ def generate_random_data(
         for ts in range(seq_length):
             z1, s1 = lif_correlation(
                 x[ts, :, :],
-                s1,
                 input_weights=input_weights,
                 recurrent_weights=recurrent_weights,
+                state=s1,
```

Comment:
I'm a bit confused by this change

Reply:
I changed the order of the parameter for `lif_correlation` because the `state` param now has a default value.